pymc3/tests/test_distributions_random.py

```diff
@@ -1380,7 +1380,6 @@ def ref_rand_evd(size, mu, evds, sigma):
                 model_args=evd_args,
             )
 
-    @pytest.mark.xfail(reason="This distribution has not been refactored for v4")
```

Comment:
The ideas is for these old tests that use `pymc3_random` to be replaced by the new, less wasteful, `BaseTestDistribution` tests. Once you confirm this passes locally, you can go ahead and replace it.